documentation/staging/content/userguide/base-images/patch-images.md

```diff
@@ -126,17 +124,7 @@ The `rebase` command does the following:
 
 Using `rebase`, the new image can be created in one of two ways:
 
-- As a new WebLogic image from a base OS image (similar to the `create` command; recommended).
-
-  To activate:
-  - Set `--tag` to the name of the final new image.
-  - Set `--sourceImage` to the WebLogic image that contains the WebLogic configuration.
-  - Set additional fields (such as the WebLogic and JDK locations),
-    similar to those used by `create`.
-    See [Create a custom base image](#create-a-custom-base-image).
-  - Do _not_ set `--targetImage`.  (When
-    you don't specify a `--targetImage`, `rebase` will use
-    the same options and defaults as `create`.)
+- As a new WebLogic image from a base OS image (similar to the `create` command; recommended). 
```

Comment:
I would restate this bullet.
As a new WebLogic image from a base OS image (similar to the WebLogic Image Tool 'create`).  Oracle strongly recommends rebasing your images and applying the latest security patches by with  the `--recommendedPatches` option.

Reply:
Done.